INSTALL.md

```diff
@@ -23,7 +23,7 @@ To build and install the tpm2-tools software the following software is required:
   * Curl library and header files
 
 #### Optional Dependencies:
-  * To build the man pages you need [md2man-roff](https://github.com/sunaku/md2man)
+  * To build the man pages you need [md2man-roff](https://github.com/sunaku/md2man) and pandoc
```

Comment:
nope just pandoc!

We might want to make INSTALL.md point to this part of the wiki or vice-versa:
https://github.com/tpm2-software/tpm2-tools/wiki/Getting-Started#installing

But that wiki installing page is way better. So we may want to copy that context to INSTALL.md and make
the wiki Installing page point to INSTALL.md file.